lib/aurora_bootstrapper/exporter.rb

```diff
@@ -11,23 +11,32 @@ def initialize( client:, prefix: "", export_bucket:, blacklisted_tables: "", whi
       @whitelisted_tables = whitelisted_tables.split(",")
       @blacklisted_fields = blacklisted_fields.split(",")
       @client             = client
+      @notifier           = Notifier.new
     end
 
     def export!
       @client.query( "set sql_mode='NO_BACKSLASH_ESCAPES'" )
+
       database_names.all? do | database_name |
         begin
           @client.query( "use `#{database_name}`" )
           database = Database.new database_name: database_name,
                                          client: @client,
                              blacklisted_tables: @blacklisted_tables,
                              whitelisted_tables: @whitelisted_tables,
-                             blacklisted_fields: @blacklisted_fields
+                             blacklisted_fields: @blacklisted_fields,
+                             export_date: @notifier.export_date
           database.export! into_bucket: @export_bucket
         rescue => e
           AuroraBootstrapper.logger.error message: "Error in database #{database_name}", error: e
         end
       end
+
+      notify!
+    end
+
+    def notify!
+      @notifier.notify
```

Comment:
what's the benefit of wrapping this in a method? Can we just call `@notifier.notify` from `export`?

Reply:
For testing

Reply:
What is the test case that we're solving for? If the test is for the notifier, can it be addressed in the notifier suite?

Reply:
Removed and found a way to test